tool/README.md

```diff
@@ -0,0 +1,67 @@
+# TeamCity Docker Images - tools
+Tools that are used within TeamCity Docker images release process.
+
+
+
+# 1. Dockerfiles & Build Configurations generator
+[TeamCity.Docker](TeamCity.Docker) is an utility application (C#) that generates Dockerfiles along with build configurations for
```

Comment:
"along with build configuration" -> "along with Kotlin DSL for build configurations" sounds better, what do you think?